apps/web/lib/webhooks/constants.ts

```diff
@@ -7,11 +7,12 @@ export const WEBHOOK_TRIGGER_EVENTS_GROUPED_BY_APP = {
     WebhookTriggerEvents.BOOKING_CANCELLED,
     WebhookTriggerEvents.BOOKING_CREATED,
     WebhookTriggerEvents.BOOKING_RESCHEDULED,
-  ] as ["BOOKING_CANCELLED", "BOOKING_CREATED", "BOOKING_RESCHEDULED"],
+    WebhookTriggerEvents.MEETING_ENDED,
+  ] as ["BOOKING_CANCELLED", "BOOKING_CREATED", "BOOKING_RESCHEDULED", "MEETING_ENDED"],
```

Comment:
```suggestion
  ] as const,
```

To get the same result